Assets/AnkrSDK/Examples/Scripts/UI/ConnectionController.cs

```diff
@@ -25,18 +25,10 @@ public class ConnectionController : MonoBehaviour
 
 		private void OnEnable()
 		{
-		#if UNITY_WEBGL && !UNITY_EDITOR
-			try
-			{
-				_connectionText.text = ConnectingText;
-				_sceneChooser.SetActive(true);
-				_loginButton.gameObject.SetActive(false);
-			}
-			catch (Exception)
-			{
-				_connectionText.text = LoginText;
-			}
-		#else
+		#if UNITY_EDITOR || UNITY_ANDROID || UNITY_IOS
```

Comment:
just say UNITY_WEBGL && !UNITY_EDITOR ?

Reply:
Maybe !UNITY_WEBGL && !UNITY_EDITOR ?

Reply:
if you had UNITY_EDITOR || UNITY_ANDROID || UNITY_IOS the opposite of this will be   UNITY_WEBGL && !UNITY_EDITOR